### PR TITLE
fix: Chevron in Select

### PR DESCRIPTION
**Pull request by dozortsev**
☝ This title will appear in the changelog - keep it meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint).

## ✅ Checklist

- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/master/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/src/less/select.less [hunk 1]
```diff
@@ -29,10 +29,25 @@
     }
   }
 
+  .dropdown-toggle.btn-input {
+    &.btn-sm {
+      padding-right: calc(var(--space-8) * 4);
```
Comment (anttio) [inline packages/css/src/less/select.less:34]: No need to do the calc() as we have spacing tokens.

```suggestion
      padding-right: var(--space-32);
```

File: packages/css/src/less/select.less [hunk 2]
```diff
@@ -29,10 +29,25 @@
...
+    }
+
+    &.btn-md {
+      padding-right: calc(var(--space-8) * 5);
```
Comment (anttio) [inline packages/css/src/less/select.less:38]:
```suggestion
      padding-right: var(--space-40);
```

File: packages/css/src/less/select.less [hunk 3]
```diff
@@ -29,10 +29,25 @@
...
+    }
+
+    &.btn-lg {
+      padding-right: calc(var(--space-8) * 6);
```
Comment (anttio) [inline packages/css/src/less/select.less:42]:
```suggestion
      padding-right: var(--space-48);
```

File: packages/css/src/less/select.less (hunk 1 above)
Comment (dozortsev) [reply packages/css/src/less/select.less:34]: thanks @anttio

**Pull request merged**